ui/k-info-panel/switch_info.kytos

```diff
@@ -1,6 +1,38 @@
 <template>
     <k-accordion>
-      <k-button :on_click="bt_state_toggle" :title="next_state"></k-button>
+      <div class="buttons">
+        <div class="button-left">
+          <k-button :on_click="bt_state_toggle" :title="next_state"></k-button>
+        </div>
+        <div class="button-right">
+          <k-button :on_click="bt_delete" title="Delete"></k-button>
+        </div>
+      </div>
+      <template v-if="show_modal">
+        <div class="modal-mask">
+          <div class="modal-wrapper">
+            <div class="modal-container">
+              <div class="modal-header">
+                <slot name="header">
+                </slot>
+              </div>
+              <div class="modal-body">
+                <slot name="body">
+                  Delete {{metadata.dpid}} Switch? 
```

Comment:
![20240129_093359](https://github.com/kytos-ng/topology/assets/1010796/6440fc0e-5916-409b-92e9-7d144fc58c3a)

The string order with dpid coming before `"Switch"` is a bit strange. `Delete Switch {{metadata.dpid}}` would read more naturally, I believe. Also, it'd be great to have here interpolation with `metadata.node_name` (analogous to what you've for a link), right?